units_cpp/src/units/headers/units/constants/planetary_densities.h

```diff
@@ -0,0 +1,46 @@
+/* This file is auto generated.
+ * DO NOT EDIT IT MANUALLY!
+ */
+
+#ifndef _PLANETARY_DENSITIES_
+#define _PLANETARY_DENSITIES_
+
+#include <units/density_units.h>
+
+
+namespace units {
+namespace planetary_densities
+{
+
+//the ratio of the mass of the sun to the volume of space it takes up
+constexpr density::KilogramsPerLiter sunDensity{1.4};
```

Comment:
are you sure it's kilograms per liter? that sounds odd...

Reply:
originally it's gram per cubic centimeter, which is equal to kilogram per liter.
I think it's okay to leave it this way

Reply:
what about changing the name of the unit scale itself to "grams per centimeter"? what is more in common use?

Reply:
I think it's used the same. besides, we don't have "cubic centimeters" scale in volume so it's easier 😛

Reply:
ok